test/utils/utils.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
To limit our public API exposure leading up to a 1.0 release, would it make sense to put this in an internal directory for now so that it doesn't become part of our 1.0 API?

We _could_ always add it into the SDK library as a non-breaking new feature after 1.0 if users would find it helpful.

Reply:
I'd like to keep this internal, so I'll move it to `test/internal/utils`.

Reply:
probably needs to be `internal/test` so that it can be imported from anywhere in the SDK CLI repo.

Reply:
^ Disregard. `test/internal` should be okay for `utils` since we only import it from within `test`.